tensorflow/contrib/losses/python/losses/loss_ops.py

```diff
@@ -633,7 +635,7 @@ def cosine_distance(
   Args:
     predictions: An arbitrary matrix.
     labels: A `Tensor` whose shape matches 'predictions'
-    dim: The dimension along which the cosine distance is computed.
+    axis: The dimension along which the cosine distance is computed.
     weights: Coefficients for the loss a scalar, a tensor of shape
       [batch_size] or a tensor whose shape matches `predictions`.
     scope: The scope for the operations performed in computing the loss.
```

Comment:
Please include dim in the docstring (the linter will complain if you don't). Can you make sure to look at the docstring of the function that the decorator returns to make sure that dim is appropriately marked as deprecated (and if not, put "(deprecated)" in its description)?

Reply:
Thanks @martinwicke. The docstring has been updated with `dim` at the last line (match order of the API). Also dim is marked as deprecated correctly:
```
ubuntu@ubuntu:~$ python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
>>> print tf.losses.cosine_distance.__doc__
Adds a cosine-distance loss to the training procedure. (deprecated arguments)

SOME ARGUMENTS ARE DEPRECATED. They will be removed in a future version.
Instructions for updating:
dim is deprecated, use axis instead

Note that the function assumes that `predictions` and `labels` are already
unit-normalized.

Args:
  labels: `Tensor` whose shape matches 'predictions'
  predictions: An arbitrary matrix.
  axis: The dimension along which the cosine distance is computed.
  weights: Optional `Tensor` whose rank is either 0, or the same rank as
    `labels`, and must be broadcastable to `labels` (i.e., all dimensions must
    be either `1`, or the same as the corresponding `losses` dimension).
  scope: The scope for the operations performed in computing the loss.
  loss_collection: collection to which this loss will be added.
  reduction: Type of reduction to apply to loss.
  dim: The old (deprecated) name for `axis`.

Returns:
  Weighted loss float `Tensor`. If `reduction` is `NONE`, this has the same
  shape as `labels`; otherwise, it is scalar.

Raises:
  ValueError: If `predictions` shape doesn't match `labels` shape, or
    `axis`, `labels`, `predictions` or `weights` is `None`.
>>> exit()
ubuntu@ubuntu:~$ 
```